web/app/stylesheet.css.sass

```diff
@@ -1,4 +1,109 @@
 @import "bootstrap"
 
-body
-  margin: 10px 0
+a
+  font-weight: 400
+  font-family: 'open_sans_regular', 'Helvetica Neue', Helvetica, Arial, sans-serif
+  text-decoration: none
+  list-styling: none
+  color: white
```

Comment:
this made the link to GitHub on the "about" page invisible. can we change that page's background so it's dark, or have alternate dark and light styles, or something like that?